open-company/open-company-web
# Always go back to AP

Card: https://trello.com/c/NKBtcRdq

To test:
- open AP
- click on a post
- [x] button says Back to All posts?
- now go to a section
- click on a post
- [x] button says Back to Section?
- now copy the link of a post
- open it
- [x] button says Back to All posts?
- now change a section to public
- copy a post url from the section
- open it in incognito
- [x] button says Back to Section (since you are not signed in you can't see AP)?
